### PR TITLE
#392 Deprecate mergeRows/mergeCols and agglomerateByRank/Prevalence

### PR DESCRIPTION
**Pull request by 0xMuluh**
Deprecate methods with a warning that they have been replaced with another name

- mergeCols -> mergeSamples
- mergeRows -> mergeFeatures
- agglomerateByRank -> mergeFeaturesByRank 
- agglomerateByPrevalence -> mergeFeaturesByPrevalence

see #392

### CODE REVIEW

**Review by antagomir — COMMENTED** [review]
Also update:
- unit tests
- vignette
- NEWS file
- DESCRIPTION file (version number)

File: R/agglomerate.R [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#' Agglomerate data using taxonomic information
+#' Ag           glomerate data using taxonomic information
```
Comment (antagomir) [inline R/agglomerate.R]: fix?

Comment (0xMuluh) [reply R/agglomerate.R]: Ack.:) Typo.

**Comment by 0xMuluh** [comment]
> Also update:
> 
> * unit tests
> * vignette
> * NEWS file
> * DESCRIPTION file (version number)

ACK.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: mia
 Type: Package
-Version: 1.9.9
+Version: 2.0.0
```
Comment (TuomasBorman) [inline DESCRIPTION]: next version is 1.9.10

**Comment by 0xMuluh** [comment]
> Also update:
> 
> * unit tests
> * vignette
> * NEWS file
> * DESCRIPTION file (version number)

ACK

File: DESCRIPTION (hunk 2 above)
Comment (0xMuluh) [reply DESCRIPTION]: ack

**Comment by antagomir** [comment]
resolve the conflicts..?

**Comment by 0xMuluh** [comment]
> resolve the conflicts..?

ack

**Comment by antagomir** [comment]
Can we implement the same name change to:

- miaViz
- miaTime
- miaSim

Including R/ files, vignettes, and unit tests.

Should be a straightfwd fix.

(if relevant; might not be used in all these packages?)

@Daenarys8 would be the most handy if you could do this on the same go?

**Comment by antagomir** [comment]
@ake123 could you update these names to OMA?

**Comment by ake123** [comment]
I tried to render the OMA book and having some trouble in updating it. Even updating it with the above changes is throwing errors.
 After syncing the repo I am having this error
```r
Quitting from lines 294-296 [dmm5] (24_clustering.Rmd)
Error:
! 'name' does not a list of DMN objects.
Backtrace:
 1. miaViz::plotDMNFit(tse_dmm, type = "laplace", name = "DMM")
 2. miaViz::plotDMNFit(tse_dmm, type = "laplace", name = "DMM")
 3. miaViz (local) .local(x, name, type, ...)
 5. mia::getDMN(x, name)
 6. mia (local) .local(x, name, ...)
 7. mia:::.get_dmn(x, name)
Execution halted
Error in Rscript_render(f, render_args, render_meta, add1, add2) : 
  Failed to compile 24_clustering.Rmd
```

**Comment by antagomir** [comment]
@BananaCancer ?

**Comment by Basil-Courbayre-Dussau** [comment]
Could I see the dmm related code being executed @ake123 ?
The OMA book passed the checks and runned well on my end last time I checked.

Maybe the latest version of miaViz isn't being used?

**Comment by ake123** [comment]
Installed the necessary dependencies to build the book before rendering using "BiocManager::install(remotes::local_package_deps(dependencies=TRUE))"

**Comment by antagomir** [comment]
Ok - and that's an OMA issue (to be discussed in github/OMA). Indeed the latest miaX pkgs need to be updated to Bioc first.. usually in that order.

**Pull request merged**